db-changes/datasets/28102021_alter_study_user.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE study_user RENAME COLUMN receive_anonymization_report TO receive_study_user_report;
```

Comment:
On qualif (maybe mysql is not up to date ? https://stackoverflow.com/questions/30290880/rename-a-column-in-mysql)
ALTER TABLE study_user RENAME COLUMN receive_anonymization_report TO receive_study_user_report; 
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'COLUMN receive_anonymization_report TO receive_study_user_report' at line 1

Reply:
Could be

`ALTER TABLE study_user CHANGE receive_anonymization_report receive_study_user_report bit(1);`

Reply:
done